src/test/config.ts

```diff
@@ -0,0 +1,3 @@
+export const config = {
```

Comment:
There is a already a config file in config/test.json

Reply:
not sure if we were gonna keep it as all the configs ive seen in hmcts repos are all yaml files and this typescript file allows me to check the environment the service is in.